.doks/Makefile

```diff
@@ -24,12 +24,16 @@ define call_helm_install
 	--set global.cacheSecret=$${SHARED_PLAY_CRYPTO_SECRET} \
 	--set codacy-api.config.license=$${CODACY_LICENSE} \
 	--set global.codacy.url=${4} \
+	--set global.codacy.crow.url=${4}/monitoring \
 	--set global.codacy.backendUrl=${4} \
 	--set listener.nfsserverprovisioner.storageClass.name="${1}-listener-cache-class" \
 	--set global.features.cloneSubmodules=true \
 	--set global.bitbucketEnterprise.consumerPublicKey=$${BITBUCKET_ENTERPRISE_PUBLIC_KEY} \
 	--set global.bitbucketEnterprise.consumerPrivateKey=$${BITBUCKET_ENTERPRISE_PRIVATE_KEY} \
-	--set worker-manager.config.codacy.worker.image=${5}
+	--set worker-manager.config.codacy.worker.image=${5} \
+	--set crow.config.crow.url=${4}/monitoring \
```

Comment:
Why do we need two crow urls?

Reply:
this has been fixed on codacy-website's side. 👍